copier/types.py

```diff
@@ -75,3 +81,34 @@ def __getitem__(self, key: str) -> Any:
         if key not in self.done:
             self.done[key] = self.pending[key]()
         return self.done[key]
+
+
+class Phase(str, Enum):
+    """The known execution phases."""
+
+    PROMPT = "prompt"
+    TASKS = "tasks"
+    MIGRATE = "migrate"
```

Comment:
Admittedly totally subjective, but would

```suggestion
    MIGRATIONS = "migrations"
```

be more consistent with

```python
TASKS = "tasks"
```

without introducing inconsistency with the other members?

Reply:
I don't really have an opinion on the enum members semantic.
They were named after the action being executed in that phase, `prompt`, `migrate` and `render` are actions. `tasks` is short for "execute tasks".
I can rename them whatever suit you.
But for consistency, if you go for `migrations`, aka. the type of object dealt in the phase, I would go for: 
- ~`prompt`~ > `questions`
- `tasks`
- ~`migrate`~ > `migrations`
- `render` > ??? (hard to describe the object there instead of the action)

Reply:
Thanks for sharing your thoughts. Let's keep the action semantics.